docs-markdown/src/test/suite/extension.test.ts

```diff
@@ -1,19 +1,28 @@
-import * as assert from 'assert';
+import * as chai from 'chai';
+import * as spies from 'chai-spies';
+import { deactivate, installedExtensionsCheck, setupAutoComplete } from '../../extension';
+import { sleep, sleepTime } from '../test.common/common';
 
-// You can import and use all API from the 'vscode' module
-// as well as import your extension to test it
-import { window } from 'vscode';
-import { deactivate, installedExtensionsCheck } from '../../extension';
-
-suite('Extension Test Suite', () => {
-	window.showInformationMessage('Start all tests.');
-
-	test('Sample test', () => {
-		deactivate();
+chai.use(spies);
+const expect = chai.expect;
 
+suite('Extension tests', () => {
+	test('installedExtensionsCheck', () => {
+		const spy = chai.spy(installedExtensionsCheck);
 		installedExtensionsCheck();
-
-		assert.equal(-1, [1, 2, 3].indexOf(5));
-		assert.equal(-1, [1, 2, 3].indexOf(0));
+		sleep(sleepTime);
+		expect(spy).to.be.spy;
```

Comment:
Im not sure I understand what is being tested here. It looks like we are checking if the function you setup as a spy is a spy? Maybe we want to check if the function has been called and call activate?

Reply:
Hi @bharney0 , I'll take a look.  It should test for calls but maybe I committed this when I was testing for external extensions.

Reply:
Hi @bharney0 , forgot to push my last commit.  Can you take a look now?  Thanks!